login.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<title>Login</title>
+</head>
+<body>
+	<!-- sectiuni cu imagini si text -->
+	<header>
+		<img src = "./assets/feedback-banner.jpg">
+		<form>
+			<input type = "name" name = "name" placeholder = "Username">
+			<input type = "password" name = "password" placeholder = "Password">
+			<input type = "submit" name = "submit" value = "Login">
+		</form>
+	</header>
+	<section style = "display:inline-flex;">
+		<img src = "./assets/new feedback.jpg">
+		<article>
+			<h1>Submit new feedback</h1>
+			<p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nulla quam velit, vulputate eu pharetra nec, mattis ac neque. Duis vulputate commodo lectus, ac blandit elit tincidunt id. Sed rhoncus, tortor sed eleifend tristique, tortor mauris molestie elit, et lacinia ipsum quam nec dui. Quisque nec mauris sit amet elit iaculis pretium sit amet quis magna. Aenean velit odio, elementum in tempus ut, vehicula eu diam. Pellentesque rhoncus aliquam mattis. Ut vulputate eros sed felis sodales nec vulputate justo hendrerit. Vivamus varius pretium ligula, a aliquam odio euismod sit amet. Quisque laoreet sem sit amet orci ullamcorper at ultricies metus viverra. Pellentesque arcu mauris, malesuada quis ornare accumsan, blandit sed diam.</p>
+		</article>
+	</section>
+	<section style = "display:inline-flex;">
+		<article>
+			<h1>Submit new feedback</h1>
```

Comment:
Well, maybe not _always_... just very, very often :stuck_out_tongue_winking_eye: